docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.1'
+
+services: 
+  db: 
+    image: postgres:9-alpine
+    restart: always
+    environment:
+      - POSTGRES_USER=wallet_user
+      - POSTGRES_PASSWORD=secret
+    ports: 
+      - 5432:5432
+    volumes: 
+      - postgres_volume:/var/lib/postgresql
+      - ./init-db.sql:/docker-entrypoint-initdb.d/init-db.sql
```

Comment:
I think this brings big convenience, thanks.
@mzyau about the init SQL, do we still are using it for set up DB? or just need to use db-migrate?

Reply:
Yup, we no longer need to seed the db with a sql file since we can use migrations now.

Reply:
Cool! I was thinking to set up docker to run migration as well, but probably can do that later as a separate thing

Reply:
@dadiorchen I've removed the init script, so I think this could be merged, unless anything else needs changing. Thanks